content/documentation/known-issues/index.adoc

```diff
@@ -28,16 +28,32 @@ No version of Internet Explorer is supported with Kiali. Users may experience so
 
 In order to have the best experience with Kiali you will need to update to use a link:../prerequisites/#_browser_requirements[supported browser].
 
-== Non Deployment-based workloads
-
-In Istio, a _Workload_ can be any pod or group of pods, regardless where they originate from. They may come from Kubernetes Deployments, Replica Sets or even as a single "orphan" pod. In Kiali, we essentially assume, currently, that a Workload comes from a Deployment. They should represent the vast majority of the cases.
-
-icon:info[size=2x] {nbsp}{nbsp}{nbsp}{nbsp} JIRA link: https://issues.jboss.org/browse/KIALI-1284[KIALI-1284]
-
 == Graph badly laid out
 
 If you search in the https://groups.google.com/forum/#!msg/kiali-dev/[mailing list], you will find a number of discussions about the graph layouts.
 
 The layout for Kiali Graph may render differently, depending on the data to display (number of graph nodes and their interactions) and it's sometimes difficult, not to say impossible, to have a single layout that renders nicely in every situations. That's why Kiali offers a choice of several layout algorithms. However, we may still find some scenarios where none of the proposed algorithm offer a satisfying display. If Kiali doesn't not render your graph layout in a satisfactory manner please switch to another layout option. This can be done from the _Graph Settings_ dropdown by changing the _Layout Schema_.
 
 icon:info[size=2x] {nbsp}{nbsp}{nbsp}{nbsp} If Kiali doesn't produce a good graph for you, don't hesitate to https://github.com/kiali/kiali/issues/new?template=bug_report.md[open an issue in GitHub] or reach us in the https://groups.google.com/forum/#!forum/kiali-users[mailing lists], and describe your situation.
+
+== Many "unknown" connections in the graph
+
+In some situations you can see a lot of connections from an "Unknown" node to your services in the graph, because some software external to your mesh must be periodically pinging or fetching data. This is typically the case when you setup Kubernetes liveness probes, or have some application metrics pushed or exposed to a monitoring system such as Prometheus. Perhaps you wouldn't like to see these connections because they make the graph harder to read.
+
+There are several ways to prevent Istio from gathering this kind of telemetry.
```

Comment:
this section sounds very Istio specific. isn't there any official documentation that can be linked to instead of maintaining this here?

Reply:
It is istio specific indeed but it's generally unseen or ignored by the users til they're watching a graph in kiali. As shown is the stack overflow link i put in description, people will identify it as an issue in kiali, even if the root cause is in istio.

I didn't find any istio doc dealing specifically with this topic.

Reply:
ok